src/Source.cpp

```diff
@@ -39,22 +38,27 @@ ProcessMessageResult Source::process_message(FlatbufferMessage const &Message) {
       }
       return ProcessMessageResult::OK;
     } catch (const HDFWriterModuleRegistry::WriterException &E) {
-      LOG(Sev::Error, "Failure while writing message: {}", E.what());
+      Logger->error("Failure while writing message: {}", E.what());
       return ProcessMessageResult::ERR;
     }
   }
   return ProcessMessageResult::ERR;
 }
 
 void Source::close_writer_module() {
+  // TODO: should log here using Source::Logger field but it's been std::moved
+  // in CommandHandler::addStreamSourceToWriterModule() and would be NULL here
```

Comment:
Using `spdlog::get()` is fine in this function as it is executed rarely. This TODO comment can be removed.